try{
    let b;
    let a = [1,2,2];
    console.log(a.length);
    console.log(b.length);
}catch(e){
    console.log("inside catch");
}
console.log("hi")